tests/robot-tests/tests/visual_testing/tables_and_charts.robot

```diff
@@ -0,0 +1,163 @@
+*** Settings ***
+Resource    ../libs/common.robot
+Resource    ../libs/charts.robot
+Library     ../libs/visual.py
+Library     tables_and_charts.py
+
+*** Variables ***
+${SNAPSHOT_FOLDER}=             snapshots/%{RUN_IDENTIFIER}
+${KEY_STATS_FOLDER}=            key-stats
+${SECONDARY_STATS_FOLDER}=      secondary-stats
+${CONTENT_SECTIONS_FOLDER}=     content-sections
+${FAST_TRACKS_FOLDER}=          fast-tracks
+
+*** Keywords ***
+Check release
+    [Arguments]    ${release}
+
+    Log to console    \n\n\t=====================================================================\n
+    Log to console    \tProcessing release at %{PUBLIC_URL}${release.url}
+
+    user navigates to public frontend    %{PUBLIC_URL}${release.url}
+
+    IF    ${release.has_key_stat_blocks} is ${TRUE}
+
+        Log to console    \n\tCapturing Key Stats:
+
+        user waits until page contains element    xpath://h2[contains(text(), "Headline facts and figures")]
```

Comment:
Just wondering if you've run into any transient failures due to not using things such as `%{WAIT_SMALL}` in these cases?

Reply:
Not during my testing no I don't believe so